### ISSUE TITLE
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

### ISSUE THREAD

**Issue by msftedad** [uxsolutions/bootstrap-datepicker#2720]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7653)
Browser: Edge (Version 144.0.3719.82)
Tool: Color Contrast Analyzer Tool
URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ

**User Impact:**
Users with low vision may struggle to identify which calendar date is focused because the focus indicator has insufficient contrast (1.2:1 instead of the required 3:1). This makes keyboard navigation and focus tracking difficult, reducing accessibility and usability.

**Repro Steps:**

1. Open the URL: https://codesandbox.io/p/devbox/epic-jasper-gz8pcz?file=%2findex.html&workspaceId=ws_MVpzSDr2nAm9FXcihgWukZ in edge browser in windows.
2. Press tab key to navigate to the 'Select Date' control.
3. Calendar will open. 
4. Press tab key to navigate to the calendar dates. 
5. Open Color Contrast Analyzer Tool.
6. Verify whether Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 or not.

**Actual Result:**
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

**Refer Attachment:**

1. Luminosity contrast ratio of focus indicator on calendar dates is less than expected ratio.png
2. Luminosity contrast ratio of focus indicator on calendar dates is less than expected ratio.mp4

https://github.com/user-attachments/assets/5156e347-2609-41f8-9202-9ff11f1fd6e0
<img width="969" height="563" alt="Image" src="https://github.com/user-attachments/assets/ec78e872-a304-4040-8b68-90bd2035e193" />

**Expected Result:**
Luminosity contrast ratio of focus indicator on calendar dates is 1.2:1 which is less than expected ratio 3:1 in windows.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, please provide an update on this issue?
